templates/repo/issue/labels/labels_selector_field.tmpl

```diff
@@ -1,4 +1,4 @@
-<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-label dropdown">
+<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-label label-filter dropdown">
```

Comment:
Don't do this. The `label-filter` styles will pollute this `select-label` dropdown.

Reply:
Thanks, changed to change selector instead in https://github.com/go-gitea/gitea/pull/25359/commits/8ec7f485ca97b474c5406c0c26e57c4a0224bc55